.travis.yml

```diff
@@ -153,7 +153,8 @@ git:
 cache:
     ccache: true
     directories:
-        - boost_1_67_0
+#        - boost_1_67_0
```

Comment:
Should it stay?

Reply:
No, all those build triggering changes will have to go.